modAionImpl/src/org/aion/zero/impl/AionHub.java

```diff
@@ -122,21 +125,35 @@ public static AionHub inst() {
     private static final int INIT_ERROR_EXIT_CODE = -1;
 
     public AionHub() {
+        initializeHub(CfgAion.inst(), AionBlockchainImpl.inst(), AionRepositoryImpl.inst(), false);
+    }
+
+    private void initializeHub(
+            CfgAion _cfgAion,
+            AionBlockchainImpl _blockchain,
+            AionRepositoryImpl _repository,
+            boolean forTest) {
```

Comment:
Not a huge fan of passing these forTest booleans around...  but can't think of any way to make this work without refactoring everything AionHub touches.  Oh well, I think it's still an improvement over how the code currently is.